### DISCUSSION TITLE
I put 'rustflags' in pyproject.toml [build]. Is that wrong?

### DISCUSSION THREAD

**Discussion by CarlKCarlK** [PyO3/maturin#1834] [Q&A] [answered]
The `pyproject.toml` for [our genomics package](https://github.com/fastlmm/bed-reader/blob/master/pyproject.toml) ends with:

```
[build]
# https://nnethercote.github.io/perf-book/compile-times.html
rustflags = ["-C", "link-arg=-fuse-ld=lld"]
```

The "Even Better TOML" linter is complaining that `[build]` is not allowed in `pyproject.toml`.

Is there a better place for rustflags?

**Comment by messense** [comment, marked as answer]
It's wrong, `rustflags` can only be set in  `.cargo/config.toml` or `~/.cargo/config.toml`.

See https://doc.rust-lang.org/cargo/reference/config.html
